PCbuild/python.vcxproj

```diff
@@ -126,6 +126,12 @@
     <Message Text="Generating $(OutDir)pybuilddir.txt" />
     <WriteLinesToFile File="$(OutDir)pybuilddir.txt" Lines="%0D%0A" Overwrite="true" />
   </Target>
+  <Target Name="GenerateBuildDetailsJSON" AfterTargets="Link">
+    <Message Text="Generating $(OutDir)build-details.json" />
+    <Exec Command='setlocal
+set PYTHONPATH=$(PySourcePath)Lib
```

Comment:
Unsure if these bits are needed, I copied them from the entry slightly further down.